packages/expo-router/html.d.ts

```diff
@@ -1 +1 @@
-export { ScrollViewStyleReset } from './build/static/html';
+export { ScrollViewStyleReset } from '@expo/router-server/build/static/html';
```

Comment:
Is it safe to have circular dependencies? From server to router and then from router to server?